### PR TITLE
Disable tagging entirely for 0.21

### PR DESCRIPTION
**Pull request by teh-cmc**
There are too many pitfalls that users can fall into because the current logging APIs are not designed with tagging in mind, and we have not yet communicated at all about any of this.

This has to be done below the chunk level, directly at the descriptor level.

### CODE REVIEW

**Comment by teh-cmc** [comment]
@rerun-bot full-check

File: crates/store/re_chunk_store/tests/gc.rs [hunk 1]
```diff
@@ -387,6 +387,8 @@ fn protected_time_ranges() -> anyhow::Result<()> {
         }
     }
 
+    eprintln!("{store}");
```
Comment (Wumpf) [inline crates/store/re_chunk_store/tests/gc.rs:390]:
```suggestion
```

debug vestigue?

**Review by Wumpf — APPROVED** [review]
🤞 hopefully that was all `.entry` callsites

File: crates/store/re_chunk_store/tests/gc.rs (hunk 1 above)
Comment (teh-cmc) [reply crates/store/re_chunk_store/tests/gc.rs:390]: On purpose -- I like the extra context when things fail.

**Comment by teh-cmc** [comment]
> 🤞 hopefully that was all `.entry` callsites

That was all the offending one for sure -- I commented out the DerefMut and all public qualifiers :+1:

**Pull request merged**